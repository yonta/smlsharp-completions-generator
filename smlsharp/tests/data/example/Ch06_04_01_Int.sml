10;
0xA;
~0xFF;

10 + 0xA;
0 - 0xA;
10 div 2;
10 * 2;

1 + 2 * 3;
4 - 3 - 2;
