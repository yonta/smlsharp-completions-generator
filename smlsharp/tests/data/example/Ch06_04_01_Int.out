val it = 10 : int

val it = 10 : int

val it = ~255 : int

val it = 20 : int

val it = ~10 : int

val it = 5 : int

val it = 20 : int

val it = 7 : int

val it = ~1 : int
